android/src/toga_android/widgets/webview.py

```diff
@@ -72,13 +72,6 @@ async def evaluate_javascript(self, javascript):
     def invoke_javascript(self, javascript):
         self.native.evaluateJavascript(str(javascript), ReceiveString())
 
-    def set_alignment(self, value):
-        # Refuse to set alignment unless widget has been added to a container.
-        # This is because this widget's setGravity() requires LayoutParams before it can be called.
-        if not self.native.getLayoutParams():
-            return
-        self.native.setGravity(Gravity.CENTER_VERTICAL | align(value))
```

Comment:
Because of the getLayoutParams issue, this code was never previously running during widget creation. It turns out WebView doesn't even have a setGravity method.

It doesn't make any sense for WebView to support alignment anyway, so I've just removed the method.